src/pages/GovPages/Proposals/ProposalCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useContext, useState } from 'react'
+import styled, { ThemeContext } from 'styled-components'
+import { Clock } from 'react-feather'
+import { Flex } from 'rebass'
+
+import { TYPE } from '../../../theme'
+import Row, { RowBetween } from '../../../components/Row'
+import { AutoColumn } from '../../../components/Column'
+import Card, { LightCard } from '../../../components/Card'
+
+import useInterval from '../../../hooks/useInterval'
+import HourGlass from '../../../assets/svg/hourglass.svg'
+import Block from '../../../assets/svg/block.svg'
+import Done from '../../../assets/svg/done_all.svg'
+
+const Container = styled(LightCard)<{ isPassed: number }>`
```

Comment:
Just a thought: I feel like in these cases using a dedicated enum can be great for clarity (something like `VoteStatus`)